test/e2e/scenario/vulnerability-overflow/00-assert.yaml

```diff
@@ -24,5 +22,7 @@ collectors:
   - type: pod
     namespace: image-scanner
     selector: control-plane=image-scanner
-  - type: command
-    command: kubectl describe node
+  - type: pod
+    namespace: image-scanner-jobs
+    selector: workload.statnett.no/name=vuln-app
+    tail: -1
```

Comment:
I think it's too much to have the entire log, only the last part is interesting.
```suggestion
```